sdk/servicebus/service-bus/samples/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@azure/service-bus-samples",
+  "version": "0.0.1",
+  "description": "@0azure/service-bus samples",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "author": "Microsoft",
+  "license": "MIT",
+  "dependencies": {
+    "@azure/ms-rest-nodeauth": "^2.0.1",
+    "@azure/service-bus": "^1.0.2",
+    "https-proxy-agent": "^2.2.1",
```

Comment:
Are we going to add all the dependencies ?

Reply:
I believe only the ones needed to run the samples.

@sadasant In `scheduledMessages.ts` sample file, we import `delay` from `rhea-promise`. Please change it to import `delay` from `@azure/service-bus` so that we can remove `rhea-promise` from this list